sw/device/lib/base/status.c

```diff
@@ -0,0 +1,128 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/status.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+const uint16_t status_module_id = 0;
+
+static const char *basename(const char *file) {
+  const char *f = file;
+  while (*f)
+    ++f;  // end of string
+  while (f > file && f[-1] != '/')
```

Comment:
Fixed (grudgingly.  our dev env is ubuntu; we best-effort for other unix-alikes (mac).  We should never encounter an alternate path separator, but I added it just in case).